app/views/tournaments/show.html.erb

```diff
@@ -1,4 +1,5 @@
 <h1><%= @tournament.name %></h1>
+<div> <a href="<%= show_pools_path @tournament %>">Show pools</a></div>
```

Comment:
Extra space after `<div>`. Also, since we're not doing markup yet, you can omit this `div` all together.